pkgs/development/libraries/science/math/cudnn/default.nix

```diff
@@ -81,31 +81,27 @@ rec {
   cudnn_8_1_cudatoolkit_11_2 = cudnn_8_1_cudatoolkit_10_2.override { cudatoolkit = cudatoolkit_11_2; };
 
   cudnn_8_1_cudatoolkit_10 = cudnn_8_1_cudatoolkit_10_2.override { cudatoolkit = cudatoolkit_10; };
-  cudnn_8_1_cudatoolkit_11 = cudnn_8_1_cudatoolkit_10_2.override { cudatoolkit = cudatoolkit_11; };
```

Comment:
Deleted from `cudnn/default.nix` but, apparently, still `inherit`-ed in `all-packages.nix`: https://hercules-ci.com/github/SomeoneSerge/nixpkgs-unfree/jobs/101

Reply:
Ah good catch! fixed in https://github.com/NixOS/nixpkgs/pull/166533/commits/5e686a3a78df40f06b0c1dc064b130fe4a9d9318